src/main.rs

```diff
@@ -44,7 +44,12 @@ fn build_config() -> Result<RunKind, String> {
         "CONFIG",
     );
     options.optflag("h", "help", "Show this message");
-    options.optopt("d", "gir-directory", "Directory for girs", "GIRSPATH");
+    options.optmulti(
+        "d",
+        "gir-directories",
```

Comment:
Elsewhere you call this girs-dirs (note the double plural). Let's make this consistent in one way or another

Reply:
Good catch!